microsoft/testsuites/libvirt/libvirt_tck_tool.py

```diff
@@ -113,22 +129,70 @@ def _install_dep(self) -> None:
     def _install(self) -> bool:
         self._install_dep()
 
+        if isinstance(self.node.os, CBLMariner):
+            # tell libvirt to run qemu as root
+            libvirt_qemu_conf = PurePath("/etc/libvirt/qemu.conf")
+            self.node.tools[Echo].write_to_file(
+                'user = "root"',
+                libvirt_qemu_conf,
+                sudo=True,
+                append=True,
+            )
+            self.node.tools[Echo].write_to_file(
+                'group = "root"',
+                libvirt_qemu_conf,
+                sudo=True,
+                append=True,
+            )
+
+            self.node.tools[Usermod].add_user_to_group("libvirt", sudo=True)
+
+            # Workaround for error:
+            #
+            # error from service: GDBus.Error:org.gtk.GDBus.UnmappedGError.Quark
+            # ._g_2dfile_2derror_2dquark.Code4: Failed to open file
+            # “/proc/2192/status”: No such file or directory
+            self.node.tools[Sed].substitute(
+                "hidepid=2",
+                "hidepid=0",
+                "/etc/fstab",
+                sudo=True,
+            )
```

Comment:
It is possible but don't think it's worth it since it is just a couple of statements. And this will probably get fixed in the future, and we'll end up removing it.